airbyte-webapp/src/components/connectorBuilder/Builder/StreamConfigView.tsx

```diff
@@ -81,13 +82,26 @@ export const StreamConfigView: React.FC<StreamConfigViewProps> = ({ streamNum })
           label="HTTP Method"
           tooltip="HTTP method to use for requests sent to the API"
         />
+        <BuilderField
```

Comment:
Seems like these are not required, should they be marked as optional in the UI?

Reply:
Yes they should, good catch